Frends.AzureBlobStorage.CreateContainer/Frends.AzureBlobStorage.CreateContainer/CreateContainer.cs

```diff
@@ -39,23 +39,18 @@ private static BlobContainerClient GetBlobContainer(Input input)
     {
```

Comment:
https://github.com/FrendsPlatform/Frends.AzureBlobStorage/blob/01ed607e85ed9b92c403631e09f26d0233e10461/Frends.AzureBlobStorage.CreateContainer/Frends.AzureBlobStorage.CreateContainer/CreateContainer.cs#L23
There is no `Error` in result object.